openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -465,7 +466,8 @@ def get(self, request, usage_key_string, *args, **kwargs):
             str(usage_key.course_key),
             str(usage_key),
             disable_staff_debug_info=True)
-        return Response(json.loads(sequence.handle_ajax('metadata', None)))
+        # todo: carla plz fix this
+        return Response(json.loads(sequence.handle_ajax('metadata', None, view=PUBLIC_VIEW)))
```

Comment:
Note to future me: I need to figure out where to differentiate between public view and student view.